using-timescaledb/hypertables.md

```diff
@@ -126,7 +126,8 @@ taken if you make heavy use of expensive index types (e.g., some
 PostGIS geospatial indexes).  During testing, you might check your
 total chunk sizes via the [`chunk_relation_size`][chunk_relation_size] function.
 
-**Space partitions**: Space partitioning is optional but can make
+**Space partitions**: [](best-practices-space-partitions)
```

Comment:
Is this (best-practices-space-partitions) because it's an anchor link?

Reply:
yup

Reply:
More specifically, it's so I can directly reference this section from the `api.md`